components/common/typescript.gni

```diff
@@ -137,8 +139,11 @@ template("transpile_web_ui") {
       "--depfile_path=" + rebase_path(depfile),
     ]
 
-    if (defined(invoker.webpack_alias)) {
-      args += [ "--webpack_alias=" + invoker.webpack_alias ]
+    if (defined(invoker.webpack_aliases)) {
```

Comment:
Now that we have better control on which JS features to bundle, I think we should not default to `chromeapp` if possible. Assuming that only webtorrent requires `chromeapp` capabilities, we could add `webpack_aliases = [ "chromeapp" ]` to `components/brave_webtorrent/BUILD.gn`, and set the default value here to be empty.

Reply:
done, I hope it will not break anything because in master we have `chromeapp` hardcoded for all components.

Reply:
It should fail at the build phase if `chromeapp` is required by some other app (maybe you can confirm by removing `chromeapp` from webtorrent `BUILD.gn`). Do we have some automated UI tests to be sure? We can otherwise define a simple test plan for the QA.

Reply:
I do not think we can check `chromeapp` in UI tests, maybe @yrliou can create a browser test that will download torrent bundle where that code will be reused and fail in runtime because now it is built successfully and doesnt work in runtime only on some torrent bundles. 
Also maybe @petemill has ideas how to check `chomeapp` requirements with easier way. The trezor lib for example failed to build if we remove `browser` alias.

Reply:
@onyb I removed `chromeapp` from `components\brave_webtorrent\extension\BUILD.gn` and it compiled fine, but  didn't work in the runtime, so I propose to return back `chromeapp` by default

Reply:
Yes default is fine. Remember this isn't about capabilities but about which JS files to alias to in order to generate the compiled bundle. We could default to either browser or chromeapp and that avoids attempting to bring in NodeJS code at least.

Reply:
thanks, pushed back default alias